--- /opt/yunohost/gplayweb/local/lib/python2.7/site-packages/ext_libs/googleplay_api/googleplay.py	2015-11-26 04:55:37.000000000 -0500
+++ googleplay.new	2015-11-26 05:04:46.260203209 -0500
@@ -14,7 +14,7 @@
 import googleplay_pb2
 import config
 
-ssl_verify=True
+ssl_verify="/etc/ssl/certs/ca-certificates.crt"
 class LoginError(Exception):
     def __init__(self, value):
         self.value = value
